src/app/hooks/ApolloProvider.tsx

```diff
@@ -29,7 +29,7 @@ export const ApolloProvider = ({ children }: { children: ReactNode }) => {
 	const auth = useAuthContext();
 	const { GATEWAY_API_ROOT } = useAppConfigContext();
 	const config = {
-		jwt: auth.egoJwt,
+		jwt: auth?.egoJwt,
```

Comment:
`useAuthContext ` should never return as undefined, the type is well defined and provided with a good default value. This `?` shouldn't be necessary.

Reply:
OK fair this was a misfire, this is the only key `auth` in our app, but the error is key `auth` in a dependency
Reverted, but don't think the Headers issue above is enough to explain the bug either... investigation continues